README.md

```diff
@@ -64,10 +64,11 @@ end
 ## Shorteners
 
 Embiggen ships with a default list of URL shortening service domains (c.f.
-[Acknowledgements](#acknowledgements)) in
-[`shorteners.txt`](https://github.com/altmetric/embiggen/blob/master/shorteners.txt)
-but as it is likely to be outdated and incomplete, you are strongly encouraged
-to supply your own via `Embiggen.configure`.
+[Acknowledgements](#acknowledgements)) in [`shorteners.txt`][SHORTENERS].
+It includes several free domains, but also 1K+ branded shortener domains.
```

Comment:
Should we provide more information here?
I was thinking about saying they come from Bitly, but since they don't open the full list, I'm not sure if we can say that...